### PR TITLE
rm flat multiplier expcetion

### PR DESCRIPTION
**Pull request by MirijaHa**
This PR deactivates the flat multiplier by setting it to 1.

### CODE REVIEW

File: R/run_stress_test.R [hunk 1]
```diff
@@ -182,7 +182,6 @@ read_and_process_and_calc <- function(args_list) {
 
   cat("-- Configuring analysis settings. \n")
 
-  flat_multiplier <- assign_flat_multiplier(asset_type = asset_type)
```
Comment (jacobvjk) [inline R/run_stress_test.R:185]: after merging master, this will also have to be removed from `run_lrisk()`

File: R/run_stress_test.R [hunk 2]
```diff
@@ -280,7 +279,7 @@ read_and_process_and_calc <- function(args_list) {
     company_technology_asset_value_at_risk(
       shock_scenario = transition_scenario,
       div_netprofit_prop_coef = div_netprofit_prop_coef,
-      flat_multiplier = flat_multiplier
```
Comment (jacobvjk) [inline R/run_stress_test.R:283]: see above re `run_lrisk()`

**Review by jacobvjk — CHANGES_REQUESTED** [review]
pls merge master and adapt `run_lrisk()` the same way as `run_stress_test()`
rest is great

File: R/run_stress_test.R (hunk 1 above)
Comment (MirijaHa) [reply R/run_stress_test.R:185]: done

**Review by jacobvjk — APPROVED** [review]
lgtm

**Pull request merged**